python/publish/publish.sh

```diff
@@ -3,11 +3,24 @@
 set -eux
 
 if [ "$DRY_RUN" == "false" ]; then
+    PUSH_CHANGES=true
     echo "Creating draft release with attached files"
-    gh release create ${VERSION} --draft --verify-tag --title ${VERSION} --notes ""
+    TITLE="PyMongo ${VERSION}"
+    gh release create ${VERSION} --draft --verify-tag --title "${TITLE}" --notes "Community notes: <link>"
     gh release upload ${VERSION} $RELEASE_ASSETS/*.*
-    gh release view ${VERSION} >> $GITHUB_STEP_SUMMARY
+    JSON="url,tagName,assets,author,createdAt"
+    JQ='.url,.tagName,.author.login,.createdAt,.assets[].name'
+    echo "\## $TITLE" >> $GITHUB_STEP_SUMMARY
+    gh release view --json $JSON --jq $JQ ${VERSION} >> $GITHUB_STEP_SUMMARY
 else
     echo "Dry run, not creating GitHub Release"
     ls -ltr $RELEASE_ASSETS
+    PUSH_CHANGES=false
 fi
+
+# Ensure a clean repo
+git clean -dffx
```

Comment:
Why git clean here? Won't this delete all the dist/* files that need to be uploaded in the next step?

Reply:
Good catch, fixed.